docs/best_practices/integration_checklist.rst

```diff
@@ -60,7 +63,8 @@ Adding Your Tool to the IUC Repository
 - Within that folder, create a corresponding branch. You might name it after the tool.
 - After you have tested your tool and are completely happy with it (per
   previous sections of this document), add your tool and all associated data,
-  then Commit the changes with ``git commit``
+  then Commit the changes with ``git commit``. Finally push your changes
+  to github with ``git push``.
```

Comment:
I'd say "it depends". It's more likely they would be doing `git checkout $branch; git push upstream` since this is geared towards non-IUC contributors, for IUC contributions (or `git push origin branchname` for personal repos).

Should we instead link out to a git learning resource?